openshift/embcess-develop.bc.json

```diff
@@ -69,7 +69,7 @@
                         "env": [
                             {
                                 "name": "DOTNET_STARTUP_PROJECT",
-                                "value": "embc-app/embc-app.csproj"
+                                "value": "embc-app/embc-app.sln"
```

Comment:
How come you are changing this extension on the Environment Variable?

Reply:
I wasn't sure if it would work, I want the build to build the solution, not the project. Will this work?